### ISSUE TITLE
Support header-less input

### ISSUE THREAD

**Issue by fsateler** [paulfitz/daff#133]
Some csv files do not have headers. It would be nice to not skip the first row, and use just the column number as header for them.

**Comment by fsateler** [comment]
Also, `--unordered` mode seems very confused with such files

**Comment by paulfitz** [comment]
Agreed that doing something sensible with CSV files that don't have headers would be nice to have.  Perhaps a flag could be added to treat CSV files as if they had an extra `A,B,C,D,...` header row, without needing too many changes to the rest of the code.
